.circleci/config.yml

```diff
@@ -805,6 +805,28 @@ jobs:
                         command: |
                             curl https://webhook.entitlementcard-test.tuerantuer.org/hooks/install-<< parameters.bundle >>-$EAK_WEBHOOK
                         name: Install package via webhook
+    notify_release:
+        docker:
+            - image: cimg/node:19.1.0
+        parameters:
+            production_delivery:
+                description: Whether builds are delivered to the production or beta lane of the play store.
+                type: boolean
+        resource_class: small
+        steps:
+            - prepare_workspace
+            - restore_environment_variables
+            - install_app_toolbelt
+            - run:
+                command: echo "export RELEASE_ID='$(app-toolbelt v0 release create all ${NEW_VERSION_NAME} ${NEW_VERSION_CODE} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --production-release << parameters.production_delivery >>)'" >> ${BASH_ENV}
+                name: Create github release
```

Comment:
You need to pass the "--production-release" flag properly as you adjusted it in the toolbelt

Reply:
hm i expected to be able to use it like this @ztefanie but looks like that its not valid in circle ci
```
<< parameters.production_delivery >>--production-release<</ parameters.production_delivery >>
```

Reply:
You have to use the "#"-Syntax:

<<#parameters.production_delivery >>--production-release<</parameters.production_delivery >>